userguide/dailyuse/waydroid.rst

```diff
@@ -0,0 +1,81 @@
+Android apps
+========================
+
+`Waydroid <https://waydro.id>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as Ubuntu Touch.
+
+.. note::
+    "Computer" refers to another device you connect your Ubuntu Touch device to (via USB here).
+    Your USB-attached computer must have ``adb`` and ``fastboot`` installed.
+
+.. note::
+    You will need to execute commands on your Ubuntu Touch device (and/or attached computer) to initialize Waydroid and install APKs.
+    You can do that by using the terminal application, but it is easier to use ``adb shell`` or :doc:`set up SSH </userguide/advanceduse/ssh>` to access your Ubuntu Touch device from your computer.
+
+
+How to set up
+-------------
+
+All Halium 9 and newer devices ship with waydroid out of the box starting Ubuntu Touch focal, if your device fits these requirements you can follow the folowing steps
+
+#. Run ``adb shell`` from your computer to get a shell on your Ubuntu Touch device (or open the terminal app on your ubuntu touch device).
```

Comment:
Maybe rephrase to "Open a shell on your Ubuntu Touch device (adb, ssh or terminal app)" or similar

Reply:
rephrased